wp-content/themes/core/components/search/Controller.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Components\search;
+
+use Tribe\Project\Templates\Factory_Method;
+
+class Controller {
+	use Factory_Method;
+
+	public function action(): string {
+		return get_home_url();
+	}
+
+	public function render_button(): void {
+		get_template_part( 'components/button/button', null, [
```

Comment:
😍

Reply:
We could also straight use this in the template itself as well?

Reply:
We _could_, yes. There's a lot going into that args array, so I wasn't comfortable just dropping it there. But maybe the template could have:

```
get_template_part( 'components/button/button', null, $controller->button_args() );
```